app/assess/templates/components/application_overviews_table.html

```diff
@@ -105,7 +105,7 @@
                 <td class="govuk-table__cell"><a class="govuk-link" href="{{url_for('assess_bp.application',application_id=overview.application_id)}}">{{overview.project_name}}</a></td>
                 <td class="govuk-table__cell">{{asset_types[overview.asset_type]}}</td>
                 <td class="govuk-table__cell">&pound;{{ "{:,.2f}".format(overview.funding_amount_requested|int|round) }}</td>
-                <td class="govuk-table__cell">{{ overview.location_json_blob.country if overview.location_json_blob else 'N/A'}}</td>
+                <td class="govuk-table__cell">{{ overview.location_json_blob.country}}</td>
```

Comment:
I still think we need the `if` statement here, as won't we get an exception if the data is not populated yet?

Reply:
No it just displays as blank (see the screenshot in the PR description above ^)

Reply:
Would have more confidence in this if we had a test and the test data included some where the field is `None`, as per https://github.com/communitiesuk/funding-service-design-assessment/pull/135#issuecomment-1381723634

Reply:
Tests to be added to the backend as part of the mapper that gets the location data:
https://github.com/communitiesuk/funding-service-design-assessment-store/pull/107/files